src/Widgets/HeadlineWidget/HeadlineWidgetComponent.tsx

```diff
@@ -28,6 +28,7 @@ provideComponent(HeadlineWidget, ({ widget }) => {
       attribute="headline"
       className={classNames.join(' ')}
       tag={tag(widget.get('level'), style)}
+      // TODO: replace `speakingUrl` with our slugify package once available
```

Comment:
* [ ] FYI Not a big fan of this type of code comment. Maybe move this to a todo list?